bitnami/clickhouse/Chart.yaml

```
@@ -33,4 +33,4 @@ maintainers:
 name: clickhouse
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/clickhouse
-version: 7.1.4
+version: 7.1.5
```

Then, please bump to version 7.2.0 because we are adding a new feature by default (IPv6)
```suggestion
version: 7.2.0
```